RevenueCatUI/Scripts/CustomerCenterCallbacks.cs

```diff
@@ -0,0 +1,157 @@
+using System;
+
+namespace RevenueCatUI
+{
+    public enum RefundRequestStatus
+    {
+        Success,
+        UserCancelled,
+        Error
+    }
+
+    public enum CustomerCenterManagementOption
```

Comment:
Should we use strings as constants instead of enums? (Since we want to avoid using enums moving forward). I guess this does mean it would be quite different from the other SDKs though... So I'm ok with this in this case I guess.

Reply:
better to make the change now. I will make it. Making them constants makes sense. Devs will only do string comparison so no need to make an enum.